image_builder/install_packages.sh

```diff
@@ -34,3 +34,5 @@ if (("${CONTAINER:-0}")); then
     dnf mark install dnf yum $(<"$INST_PKGS_FP")
     dnf autoremove -y
 fi
+
+install_automation_tooling
```

Comment:
I don't see anything in the commit message about why this was added

Reply:
Good catch...now I'm wondering that myself.  It may have leaked in from the other PR.  I'll check and fix the commit message or remove it.